project/dexter-monitor/routes/user.js

```diff
@@ -204,4 +204,144 @@ exports.getUserCountByProjectName = function(req, res) {
             log.error(err);
             res.send({status:"fail", errorMessage: err.message});
         });
+};
+
+function loadUserStatusList() {
+    const sql = "SELECT * FROM DexterUserList";
```

Comment:
It is an outdated file.
Already changed to `"SELECT userId, dexterYn FROM DexterUserList"`.